sormas-backend/src/main/java/de/symeda/sormas/backend/visit/VisitFacadeEjb.java

```diff
@@ -308,7 +308,7 @@ public void deleteVisit(String visitUuid) {
 		CriteriaQuery<VisitIndexDto> cq = cb.createQuery(VisitIndexDto.class);
 		Root<Visit> visit = cq.from(Visit.class);
 		Join<Visit, Symptoms> symptoms = visit.join(Visit.SYMPTOMS, JoinType.LEFT);
-		Join<Object, Object> visitUser = visit.join(Visit.VISIT_USER);
+		Join<Object, Object> visitUser = visit.join(Visit.VISIT_USER, JoinType.LEFT);
```

Comment:
Please use appropriate Objects for this joining. 
(IE: Join<Visit, User> instead of Join<Object, Object>)